dj/api/main.py

```diff
@@ -26,13 +26,6 @@
 )
 from dj.api.graphql.main import graphql_app
 from dj.errors import DJException
-from dj.models.catalog import Catalog
-from dj.models.column import Column
-from dj.models.database import Database
-from dj.models.engine import Engine
-from dj.models.node import NodeRevision
-from dj.models.query import Query
-from dj.models.table import Table
```

Comment:
I can't remember the exact reason mentioned, but I vaguely remember @betodealmeida explaining why these seemingly unused imports needed to be here for `main.py`.

Reply:
Good to know, I added them back.